examples/gallery/embellishments/colorbar.py

```diff
@@ -41,10 +44,11 @@
     scale=100,
 )
 
+# ============
 # Create a colorbar suitable for surface topography - oleron
 fig.colorbar(
     cmap="oleron",
-    # Colorbar position justified outside map frame (J) at Middle Right (MR),
+    # Colorbar placed outside figure frame (J) at Middle Right (MR),
```

Comment:
In my opinion "figure" is not correct here, since the whole object is the figure, including the area outside the map frame where the colorbar is placed.

Reply:
I see your point. My intention to change it was that the generated figure is no geographical map. Maybe "plot" would serve as a more general term? But I am not sure whether there is a clear differentiation between "plot" and "figure".  Thus, I changed it back to "map" in commit  https://github.com/GenericMappingTools/pygmt/pull/2395/commits/29a7bb76eed7636ee74d126f54327be04d0040f1.